rust/cymbal/src/frames/mod.rs

```diff
@@ -15,17 +17,21 @@ pub mod resolver;
 // We consume a huge variety of differently shaped stack frames, which we have special-case
 // transformation for, to produce a single, unified representation of a frame.
 #[derive(Debug, Deserialize, Serialize, Clone)]
-#[serde(untagged)]
+#[serde(tag = "platform")]
 pub enum RawFrame {
+    #[serde(rename = "python")]
+    Python(RawPythonFrame),
+    #[serde(rename = "web:javascript")]
```

Comment:
Is `rename` the right macro here?

Reply:
Yep - it's a little confusing (and not documented very well), but what you're "renaming" is the enum variant name (so `Python` or `Javascript` in this case), which is what is put in the tag attribute (so platform in this case) - the tests wouldn't pass if it wasn't correct 😉